### DISCUSSION TITLE
it's possible to add custom params of jwt?

### DISCUSSION THREAD

**Discussion by mojerro** [ory/kratos#2310] [Q&A] [answered]
My backend get some custom information from jwt, such as the user's auth. I cannot find a way to add such titles.

**Comment by vinckr** [comment, marked as answer]
Hello @mojerro 
Sorry you're having trouble. We'll need more information in order to figure out what's happening here.
Can you provide some additional information, such as what were you trying to do and how are you running Ory Kratos?
Have a look at this comment&discussion it might provide some insight:
https://github.com/ory/kratos/discussions/1170#discussioncomment-561825

**Comment by mojerro** [comment]
I want to use kratos with hasura graphql-engine, and it needs some custom titles in jwt to get the auth. such as:
```
{
  iss: xxx,
  sub: xxx,
  ext: xxx,
  myCustom: xxx,
}
```

I'm not sure does kratos support change jwt's structure

**Reply by vinckr** [reply]
Check out [Ory Oathkeeper](https://github.com/ory/oathkeeper) for that!
